pkg/cloudprovider/aws/instancetype.go

```diff
@@ -155,7 +156,9 @@ func (i *InstanceType) Overhead() v1.ResourceList {
 			if cpu < cpuRange.end {
 				r = float64(cpu - cpuRange.start)
 			}
-			overhead.Cpu().Add(*resource.NewMilliQuantity(int64(r*cpuRange.percentage), resource.DecimalSI))
+			cpuOverhead := overhead[v1.ResourceCPU]
+			cpuOverhead.Add(*resource.NewMilliQuantity(int64(r*cpuRange.percentage), resource.DecimalSI))
+			overhead[v1.ResourceCPU] = cpuOverhead
```

Comment:
This resource library has bitten us before.

Possible to one-line? 
```
overhead[v1.ResourceCPU] = overhead.CPU().Add(*resource.NewMilliQuantity(int64(r*cpuRange.percentage), resource.DecimalSI))
```

Reply:
we can one-line it using our `resources.Merge` but then it's longer... so idk but the example you put wouldn't work because Add() doesn't return a Quantity, it just updates the pointer :(